efficientdet/keras/train.py

```diff
@@ -74,7 +74,7 @@
 flags.DEFINE_integer('batch_size', 64, 'training batch size')
 flags.DEFINE_integer('eval_samples', 5000, 'The number of samples for '
                      'evaluation.')
-flags.DEFINE_integer('steps_per_execution', 1000,
+flags.DEFINE_integer('steps_per_execution', 1,
```

Comment:
Disable it in default since there is some issues in multi gpus training with uninitlized optimizer.

Reply:
sgtm.